bin/fix_my_workspace

```diff
@@ -4,8 +4,17 @@ $VERBOSE = nil  # Don't care about world writable dir warnings and the like
 if $_fix_my_workspace_version_check
   $_fix_my_workspace_version = '0.5.0'
 else
-  require 'origen'
-
+  # if we're running this from a development environment (i.e., bin/ inside of origen repo) then
```

Comment:
This needs to be changed upstream in the Origen updater plugin, once an app updates to the newer version that plugin it will automatically mirror the changes to the local copy of fix_my_workspace like this one.